apis/v1alpha1/training_job.go

```diff
@@ -67,8 +67,8 @@ type TrainingJobStatus struct {
 // TrainingJob is the Schema for the TrainingJobs API
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="TrainingJobStatus",type=string,JSONPath=`.status.trainingJobStatus`
```

Comment:
can you revert changes which are specifically related to ordering changes of the printer columns? in here and crds?

this needs to be fixed but until then we can take care of it manually so that every PR doesn't try to change this